app/Modules/Availability/AvailabilityMinimumTimeException.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Modules\Availability;
+
+use Exception;
+use Nuwave\Lighthouse\Exceptions\RendersErrorsExtensions;
+
+class AvailabilityMinimumTimeException extends Exception implements RendersErrorsExtensions
+{
+    protected $message = 'The minimum availability time is 30 minutes';
```

Comment:
This message should be tied to the variable defined in the service

Reply:
Yeah, i tried to interpolate it but couldn't remember how to do it, even tho I did it in the other exception 😃

Reply:
done